### ISSUE TITLE
Auto-refresh build logs

### ISSUE THREAD

**Issue by g4s8** [yegor256/rultor#1244]
Let's add auto-refrsh feature for build logs, like in Travis

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@g4s8 but it works exactly that way now. If you open log page, the text will be loaded in runtime, coming from the server. Or it doesn't?

**Comment by g4s8** [comment]
@yegor256 no, at moment you need to refresh the page manually to see new logs

**Comment by 0crat** [comment]
Job [#1244](https://github.com/yegor256/rultor/issues/1244) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1244](https://github.com/yegor256/rultor/issues/1244) assigned to @filfreire[/z](https://www.0crat.com/u/filfreire), here is [why](https://www.0crat.com/footprint/C3SAYRPH9/431652125). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
The user @filfreire[/z](https://www.0crat.com/u/filfreire) resigned from [#1244](https://github.com/yegor256/rultor/issues/1244), please stop working

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 points just awarded to @filfreire[/z](https://www.0crat.com/u/filfreire), total is [+525](https://www.0crat.com/u/filfreire)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by yegor256** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/rultor/issues/1244#issuecomment-383536164))

@yegor256 GitHub issue is still open, won't close

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@0crat out

**Comment by llorllale** [comment]
@yegor256 I'm wondering why you want to exclude this feature request from scope? Would be nice to have...
